jib-maven-plugin/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file.
 
 - `jib:buildTar` goal to build an image tarball at `target/jib-image.tar`, which can be loaded into docker using `docker load` ([#514](https://github.com/GoogleContainerTools/jib/issues/514))
 
+- For Docker Hub, also tries registry aliases when getting a credential from `docker.config`
```

Comment:
nit: probably just call it "the Docker config" - `docker.config` may imply something else
Also, for formatting, it's good to include the pull request/issue number/link and I think this line should probably be under `Fixed`

Reply:
+1, also adding this to `jib-gradle-plugin`.